docs/friends.md

```diff
@@ -0,0 +1,100 @@
+
+!!! success "Fun fact"
+    Nakama is a common Japanese word that directly translates to friend or comrade. Some believe the word means "people who are considered closer than family", though that is not a part of the official definition. We feel it expresses the kind of social communities we want developers to build into their games and apps!
+
+Any social community must be maintained carefully to prevent spam or abuse. To help with this problem it's also possible for a user to block users they no longer want to communicate with and for the server to ban a user via server-side code to completely disable an account.
+
+## Add friends
+
+A user can add one or more friends by that user's ID or handle. The user added will not be marked as a friend in the list until they've confirmed the friend request. The user who receives the request can confirm it by adding the user back.
+
+!!! hint
+    A user who registers or links their account with Facebook or another social network will have friends from that network be added automatically into their friend list.
+
+When a friend request is sent or the user is added an in-app notification will be sent. See the [in-app notification](in-app-notifications.md#receive-notifications) section for more info.
+
+```csharp fct_label="Unity"
+byte[] userId = ...; // some user ID
+var message = NFriendAddMessage.Default(userId);
+client.Send(message, (bool done) => {
+  Debug.Log("Friend added or request sent.");
+}, (INError err) => {
+  Debug.LogErrorFormat("Error: code '{0}' with '{1}'.", err.Code, err.Message);
+});
+```
+
+When both users have added eachother as friends they are able to initiate realtime chat in a 1-on-1 channel. See the [realtime chat](realtime-chat.md) section for more info.
+
+## List friends
+
+You can list all of a user's friends, blocked users, friend requests received (invited), and invites they've sent. These statuses are returned together as part of the friend list which makes it easy to display in a UI.
+
+```csharp fct_label="Unity"
+var message = NFriendsListMessage.Default();
+client.Send(message, (INResultSet<INFriend> list) => {
+  foreach (var f in list.Results) {
+    var id = Encoding.UTF8.GetString(f.Id); // convert byte[].
+    // f.State is one of: Friend, Invite, Invited, Blocked.
+    Debug.LogFormat("User {0} has state {1}.", id, f.State);
+  }
+}, (INError err) => {
+  Debug.LogErrorFormat("Error: code '{0}' with '{1}'.", err.Code, err.Message);
+});
+```
+
+## Remove friends
+
+A user can remove a friend, reject a received invite, cancel a friend request sent, or unblock a user. Similar to how Friend Add works we reuse Friend Remove to cancel or undo whatever friend state is current with another user.
+
+!!! note
+    If a user is unblocked they are removed from the friend list entirely. To re-add them each user must add the other again.
```

Comment:
I misread Alim's suggestion. I think you are right Chris, no need to make this change :)